zerver/views/streams.py

```diff
@@ -141,11 +141,11 @@ def remove_default_stream(request, user_profile, stream_name=REQ()):
 
 @require_realm_admin
 @has_request_variables
-def update_stream_backend(request, user_profile, stream_id,
-                          description=REQ(validator=check_string, default=None),
-                          is_private=REQ(validator=check_bool, default=None),
-                          new_name=REQ(validator=check_string, default=None)):
-    # type: (HttpRequest, UserProfile, int, Optional[Text], Optional[bool], Optional[Text]) -> HttpResponse
+def update_stream_backend(request: HttpRequest, user_profile: UserProfile, stream_id: int,
+                          description: Optional[Text]=REQ(validator=check_string, default=None),
+                          is_private: Optional[bool]=REQ(validator=check_bool, default=None),
+                          new_name: Optional[Text]=REQ(validator=check_string, default=None),
+                          ) -> HttpResponse:
```

Comment:
Same as the comment above, move `) -> HttpResponse:` to the end of the previous line! :)

Reply:
I've tried to move it back, but found that the line was one or two characters too long for the linter to tolerate. I've tried to do something similar to line 178 and instead move the `) -> HttpResponse:` to the start of the line, although this might not be ideal

Reply:
Would this be preferable?
```python
def update_stream_backend(
        request: HttpRequest, user_profile: UserProfile,
        stream_id: int,
        description: Optional[Text]=REQ(validator=check_string, default=None),
        is_private: Optional[bool]=REQ(validator=check_bool, default=None),
        new_name: Optional[Text]=REQ(validator=check_string, default=None)
) -> HttpResponse:
```